src/notifications/index.ts

```diff
@@ -212,9 +217,55 @@ export async function generateAndStoreNotificationsV2(
   if (!args) {
     return;
   }
+  const filteredArgs = [];
+
+  for (const arg of args) {
+    const { type, ctx } = arg;
+    if (!ctx.initiatorId) {
+      filteredArgs.push(arg);
+      continue;
+    }
+
+    const userIdChunks: string[][] = [];
+    for (let i = 0; i < ctx.userIds.length; i += 500) {
+      userIdChunks.push(ctx.userIds.slice(i, i + 500));
```

Comment:
Max size of parameters we can pass to postgresql is 60k. Just if you guys want to increase the batching, although I have nothing against with 500 units.

Reply:
It is also the fact that it is a large query on a large table, but good to note about exact max 👍